src/pivotal-ui/components/forms/stepper.js

```diff
@@ -24,7 +24,7 @@ FormStepper.prototype.set = function(number){
 
   if( number <= this.min ){
     number = this.min;
-  } else if (number >= this.max ){
+  } else if (this.max && number >= this.max ){
```

Comment:
Ooo.. however, in the constructor, an undefined max would lead to `NaN` being the value here. I can check if `isNaN`